src/aat/resources/application.yml

```diff
@@ -73,4 +73,7 @@ oidc:
   issuer: ${OIDC_ISSUER_BASE_URI:http://localhost:8080/openam/oauth2/hmcts}
 
 toggles:
-  cdam_enabled: ${ENABLE_CDAM:false}
\ No newline at end of file
+  cdam_enabled: ${ENABLE_CDAM:false}
+
+endpoint-toggles:
+  search-markups: ${ENABLE_SEARCH_MARKUPS_ENDPOINT:true}
```

Comment:
@MichaelJacob21012  - Could you set this to false? Let it by default be set to false.